src/app/shared/RelayCardActions.svelte

```diff
@@ -24,12 +28,75 @@
   ]
 </script>
 
-{#if !userRelayUrls.includes(url)}
-  <button class="flex items-center gap-3 text-neutral-100" on:click={join}>
-    <i class="fa fa-right-to-bracket" /> Join
+<div class="hidden gap-4 md:flex">
+  <button
+    class="flex items-center rounded-md bg-tinted-100-l px-6 py-1 text-sm font-bold capitalize text-tinted-700-d"
+    class:bg-tinted-600-d={details}
+    class:text-tinted-100-l={details}
+    on:click={_ => (details = !details)}>
+    INFO
   </button>
-{:else if userRelayUrls.length > 1}
-  <button class="flex items-center gap-3 text-neutral-100" on:click={leave}>
-    <i class="fa fa-right-from-bracket" /> Leave
+  <button
+    class="flex items-center rounded-md bg-tinted-100-l px-6 py-1 text-sm font-bold capitalize text-tinted-700-d"
+    on:click={() =>
+      router.at(router.at("relays").of(encodeURIComponent(url)).toString()).push({modal: true})}>
+    EXPLORE
   </button>
-{/if}
+  {#if !userRelayUrls.includes(url)}
+    <button
+      class="flex items-center rounded-md bg-accent px-6 py-1 text-sm font-bold capitalize text-white"
+      on:click={join}>
+      JOIN
```

Comment:
[nitpick] Prefer typing these in normal case and using `uppercase` to transform them.

Reply:
right I used the wrong class name, capitalize rather than uppercase.